wtv-mail/Mail.html

```diff
@@ -68,11 +68,11 @@ <h2>Mail list for SKCro</h2>
       </tr>
       <tr class="mail-message clickable" role=button onClick="linkHandler('readMail.html');">
         <td><button class="message-outline clickable"></button></td>
-    	<td class=from>johnlandlord@aol.com</td>
-    	<td class=subject>Notice of Eviction</td>
+    	<td class=from>johntheboss@webtv.net</td>
+    	<td class=subject>Notice of Paycheck</td>
```

Comment:
Why change the humor here? The eviction notice is funnier.

Reply:
Felt the need

Reply:
There was literally no point in doing that. It's funny to other people - just because you don't find it funny doesn't mean you get to remove it.